File: python/004.py
#!/usr/bin/env python

import sys
import string

col1 = open("col1.txt", "Ur").read().split("\n")
col2 = open("col2.txt", "Ur").read().split("\n")

for i, r in enumerate(col1):
    print r, "\t", col2[i]

File: ruby/004.rb
#!/usr/local/bin/ruby19

r = Array.new
r[0] = open( "col1.txt", "r" ).read.split( "\n" )
r[1] = open( "col2.txt", "r" ).read.split( "\n" )

r[0].size.times do |i|
  print "#{r[0][i]}\t#{r[1][i]}\n"
end

#r[0].zip( r[1] ).each do |a,b|
#  print "#{a}\t#{b}\n"
#end

# Those two methods seem to be almost same.
